src/components/list/list.ts

```diff
@@ -56,6 +63,16 @@ export class MdListItem implements AfterContentInit {
 
   constructor(private _renderer: Renderer, private _element: ElementRef) {}
 
+  /** @internal */
+  handleFocus() {
+    this.hasFocus = true;
+  }
+
+  /** @internal */
+  handleBlur() {
+    this.hasFocus = false;
+  }
+
```

Comment:
Added.

On Mon, May 23, 2016 at 3:42 PM, Jeremy Elbourn <notifications@github.com>
wrote:

> In src/components/list/list.ts
> <https://github.com/angular/material2/pull/502#discussion_r64302059>:
>
> > @@ -56,6 +63,16 @@ export class MdListItem implements AfterContentInit {
> >
> >    constructor(private _renderer: Renderer, private _element: ElementRef) {}
> >
> > +  /** @internal */
> > +  handleFocus() {
> > +    this.hasFocus = true;
> > +  }
> > +
> > +  /** @internal */
> > +  handleBlur() {
> > +    this.hasFocus = false;
> > +  }
> > +
>
> Unit test for focus class?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/material2/pull/502/files/bbda787d2d2385175e953d24ba093c5ca3dd51ed#r64302059>
>